assets/xml/objects/object_mm.xml

```diff
@@ -7,7 +7,7 @@
         <Animation Name="gRunningManSprintAnim" Offset="0x0468"/>
         <Animation Name="gRunningManExcitedAnim" Offset="0x73A0"/>
         <Animation Name="gRunningManHappyAnim" Offset="0x8060"/>
-        <Texture Name="gRunningManMouthClosedTex" Outname="running_man_mouth_closed" Format="ci8" Width="32" Height="16" Offset="0xE30"/>
-        <Texture Name="gRunningManMouthOpenTex" Outname="running_man_mouth_closed" Format="ci8" Width="32" Height="16" Offset="0xC30"/>
+        <Texture Name="gRunningManMouthClosedTex" OutName="running_man_mouth_open" Format="ci8" Width="32" Height="16" Offset="0xE30"/>
```

Comment:
Which one is it? One them is obviously wrong. Might as well fix it here.

Reply:
Fixed.